napari/utils/notifications.py

```diff
@@ -251,13 +251,9 @@ def install_hooks(self):
         threading.excepthook to display any message in the UI,
         storing the previous hooks to be restored if necessary.
         """
-        if getattr(threading, 'excepthook', None):
-            # TODO: we might want to display the additional thread information
-            self._originals_thread_except_hooks.append(threading.excepthook)
-            threading.excepthook = self.receive_thread_error
-        else:
-            # Patch for Python < 3.8
-            _setup_thread_excepthook()
```

Comment:
Please check if this function is used somewhere. Otherwise removie it also

Reply:
good point, removed!